lib/fileProcessor.js

```diff
@@ -0,0 +1,59 @@
+const moment = require('moment');
+class FileProcessor {
+  constructor(store, files) {
+    this.store = store;
```

Comment:
No, but this creates consistency with the constructor for gitHubProcessor, which seems logical. I can remove it though.